app/views/providers/application_merits_task/statement_of_cases/show.html.erb

```diff
@@ -66,11 +66,12 @@
           <%= form.govuk_file_field :original_file, label: { text: t("generic.attach_file"), size: "m" },
                                                     hint: { text: t(".size_hint") },
                                                     classes: ["govuk-file-upload moj-multi-file-upload__input"] %>
-          <%= form.submit(
+          <%= form.govuk_submit(
                 t("generic.upload"),
                 id: "upload",
                 name: "upload_button",
-                class: "govuk-button govuk-button--secondary form-button moj-multi-file-upload__button",
+                value: "Upload",
+                class: "govuk-button--secondary moj-multi-file-upload__button",
```

Comment:
I think you should be able to [pass `secondary: true` as an option](https://govuk-form-builder.netlify.app/form-elements/submit/#generating-a-secondary-submit-button) instead of specifying the class 👍